soh/soh/Enhancements/randomizer/3drando/location_access.cpp

```diff
@@ -260,7 +259,7 @@ void RegionTable_Init() {
   areaTable[RR_ROOT] = Region("Root", "", RA_LINKS_POCKET, NO_DAY_NIGHT_CYCLE, {}, {
                   //Locations
                   LOCATION(RC_LINKS_POCKET,       true),
-                  LOCATION(RC_TRIFORCE_COMPLETED, logic->CanCompleteTriforce),
+                  LOCATION(RC_TRIFORCE_COMPLETED, logic->GetSaveContext()->triforcePiecesCollected >= ctx->GetOption(RSK_TRIFORCE_HUNT_PIECES_REQUIRED).Value<uint8_t>();),
```

Comment:
This will appear as `GetSaveContext()->triforcePiecesCollected >= ctx->GetOption(RSK_TRIFORCE_HUNT_PIECES_REQUIRED).Value<uint8_t>()` in the logic preview so I'd at least add `GetSaveContext()->` & `.Value<uint8_t>()` to `CleanCheckConditionString`